src/components/modal/MarkerModal.tsx

```diff
@@ -23,53 +52,42 @@ const MarkerModal = ({ openModal, onClose, item, ageKey }: any) => {
         <span style={{ fontSize: '24pt' }}>{item.name}</span>
       </div>
       {item.aboveAverage ? (
-        <div className="flex flex-col justify-center items-center p-3 text-white w-[100%] my-5">
-          {/* ------------------------------------------------ */}
-
-          <div className="flex flex-col justify-center items-center p-2 bg-secondary rounded-t-lg w-2/3">
-            <div className="relative">
-              <h1 className="text-3xl relative">SUPERIOR</h1>
-            </div>
-            <div className={styles.rangeValues}>
-              {item.superior} {item.units}
-            </div>
-          </div>
-          {/* ------------------------------------------------ */}
-          <div className="flex flex-col justify-center items-center p-2 bg-blue-800 w-2/3">
-            <div className="relative">
-              <h1 className="text-3xl relative">ABOVE AVERAGE</h1>
-            </div>
-
-            <div className={styles.rangeValues}>
-              {item.aboveAverage} {item.units}
-            </div>
-          </div>
-          {/* ------------------------------------------------ */}
-          <div className="flex flex-col justify-center items-center p-2 bg-neutral rounded-b-lg w-2/3">
-            <div className="relative">
-              <h1 className="text-3xl relative">AVERAGE</h1>
-            </div>
-            <div className={styles.rangeValues}>
-              {item.average} {item.units}
-            </div>
-          </div>
-        </div>
+        <BarChart chartData={data} />
       ) : (
-        <div className="flex flex-col justify-center items-center p-2 bg-accent rounded-lg w-2/3 text-white my-5">
+        <div className="flex flex-col justify-center items-center p-2 text-neutral rounded-xl w-1/2 border-2 border-neutral my-5">
           <div className="relative">
-            <h1 className="text-3xl relative">AVERAGE</h1>
+            <h1 className="text-3xl">AVERAGE</h1>
           </div>
 
-          <div className={styles.rangeValues}>
+          <div
+            style={{
+              fontFamily: 'Verdana',
+              color: 'rgb(117, 195, 117)',
+              fontSize: '16pt',
+              fontWeight: 'bold',
+            }}
+            className="text-white"
+          >
             {item.average} {item.units}
           </div>
         </div>
       )}
+
       <Accordion item={item} />
       <div className="flex gap-4 justify-center items-center border-2 p-3 my-2 w-full">
         <p style={{ fontFamily: 'Arial' }}>VideoThumb</p>
-        <a href="https://www.youtube.com/watch?v=nVsWUtN7NXI" target="_blank">
-          <img src={`src/images/${item.videoThumb}`}></img>
+        <a href={item.videoURL} target="_blank">
+          {/* <img
```

Comment:
Good call to use the `Image` tag from Next to optimize the image elements. Let's remove this comment code since we won't need it going forward.